bare-metal/container-linux/kubernetes/fl/flatcar-linux-install.yaml.tmpl

```diff
@@ -0,0 +1,47 @@
+---
+systemd:
+  units:
+    - name: installer.service
+      enable: true
+      contents: |
+        [Unit]
+        Requires=network-online.target
+        After=network-online.target
+        [Service]
+        Type=simple
+        ExecStart=/opt/installer
+        [Install]
+        WantedBy=multi-user.target
+    # Avoid using the standard SSH port so terraform apply cannot SSH until
+    # post-install. But admins may SSH to debug disk install problems.
+    # After install, sshd will use port 22 and users/terraform can connect.
+    - name: sshd.socket
+      dropins:
+        - name: 10-sshd-port.conf
+          contents: |
+            [Socket]
+            ListenStream=
+            ListenStream=2222
+storage:
+  files:
+    - path: /opt/installer
+      filesystem: root
+      mode: 0500
+      contents:
+        inline: |
+          #!/bin/bash -ex
+          curl --retry 10 "${ignition_endpoint}?{{.request.raw_query}}&os=installed" -o ignition.json
+          flatcar-install \
```

Comment:
Hmm, I guess template in coreos vs flatcar in the common container-linux-install.yaml.tmpl (still need to think about renaming the file with all the RH news and shifting names). If the intention is to slowly convert all instances of core to flatcar within the OS, that feels like its going to be painful.